sotopia/ui/streamlit_ui/rendering/render_episode.py

```diff
@@ -92,6 +92,92 @@ def rendering_episode(episode: EpisodeLog) -> None:
             )
 
 
+def rendering_episode_full(episode: EpisodeLog) -> None:
```

Comment:
instead of writing a whole new function? can we combine with the previous function?